ui/app/components/job-status/failed-or-lost.hbs

```diff
@@ -9,14 +9,28 @@
       <FlightIcon @name="info" />
     </span>
   </h4>
-  <ConditionalLinkTo
-    @condition={{this.shouldLinkToAllocations}}
-    @route="jobs.job.allocations"
-    @model={{@job}}
-    @query={{hash status=(concat '["failed", "lost", "unknown"]') version=(concat '[' @job.latestDeployment.versionNumber ']')}}
-    @label="View Allocations"
-    @class="failed-or-lost-link"
-  >
-    {{@allocs.length}}
-  </ConditionalLinkTo>
+  {{#if (eq @title "Rescheduled")}}
+    <ConditionalLinkTo
+      @condition={{this.shouldLinkToAllocations}}
+      @route="jobs.job.allocations"
+      @model={{@job}}
+      @query={{hash scheduling=(concat '["has-been-rescheduled"]') version=(concat '[' @job.latestDeployment.versionNumber ']')}}
+      @label="View Allocations"
+      @class="failed-or-lost-link"
+    >
+      {{@allocs.length}}
+    </ConditionalLinkTo>
+  {{/if}}
+  {{#if (eq @title "Restarted")}}
+    <ConditionalLinkTo
+      @condition={{this.shouldLinkToAllocations}}
+      @route="jobs.job.allocations"
+      @model={{@job}}
+      @query={{hash scheduling=(concat '["has-been-restarted"]') version=(concat '[' @job.latestDeployment.versionNumber ']')}}
```

Comment:
What is being concatened in `(concat '["has-been-rescheduled"]')` and `(concat '["has-been-restarted"]')`? It seems like it's just a string?

Reply:
Good catch; this was an artefact from earlier! Will fix.